src/views/layouts/_angulardirectives.php

```diff
@@ -147,16 +145,24 @@
         <div class="imageupload-preview" ng-show="imageinfo != null">
             <div class="imageupload-preview-sizer"></div>
             <img ng-src="{{thumb.source}}" ng-show="imageinfo != null" class="imageupload-preview-image" />
-            <div class="imageupload-infos">
-                <div class="imageupload-size" ng-show="!imageLoading">{{ imageinfo.resolutionWidth }} x {{ imageinfo.resolutionHeight }}</div>
+
```

Comment:
This changes has nothing do to with your PR, you should make PR only about a single topic. Please reset this file to master.